apps/registration/src/utils/__tests__/fhirPatientMapper.test.ts

```
@@ -0,0 +1,288 @@
+import {
+  buildFhirPatientResource,
+  mapGenderToFhir,
+  type FhirPatientResource,
```

ESLint

[error] 'FhirPatientResource' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] `@bahmni/services` type import should occur before import of `../fhirPatientMapper`

(import/order)